src/utils/chat-gpt.ts

```diff
@@ -18,13 +18,22 @@ async function askGPTChat(messages, debug) {
             logger.info('-----Question-----');
             logger.info(stringifyJSON(messages));
             logger.info('');
+            logger.info('-----OpenAI Config-----');
+            logger.info(
+                stringifyJSON({
+                    model: process.env.GPT_MODEL ?? 'gpt-4',
+                    // Adjust this value (0 to 1) to control randomness. Lower values make the output more deterministic.
+                    // https://platform.openai.com/docs/guides/gpt/how-should-i-set-the-temperature-parameter
+                    temperature: 0.2,
```

Comment:
Move this to a variable or config higher in the file that can be shared between functions.